frontend/src/App.vue

```diff
@@ -8,5 +8,10 @@
 </template>
 
 <script setup lang="ts">
+import axios from 'axios'
 import FooterComponent from './components/FooterComponent.vue'
+import router from './router'
+axios.get(`${import.meta.env.VITE_API_URL}/projects`).then(() => {
+  router.push({ path: '/project' })
```

Comment:
We have no such endpoint. Should parse output from `projects/` and jump to the last one